services/webnn/ort/ort_model_editor.cc

```diff
@@ -15,7 +16,49 @@ namespace webnn {
 
 namespace ort {
 
-OrtModelEditor::ModelInfo::ModelInfo() = default;
+OrtModelEditor::WeightDeleter::WeightDeleter() {
+  const OrtApi* ort_api = GetOrtApi();
+  CHECK(IsSuccess(ort_api->CreateCpuMemoryInfo(
+      OrtDeviceAllocator, OrtMemTypeDefault,
+      ScopedOrtMemoryInfo::Receiver(cpu_memory_info).get())));
+
```

Comment:
It might be worth explaining OrtAllocator is a C-style structure (with a pointer to its definition) and Info, Free are just function pointers that need to be set to lambda functions.